src/ec/packages/ec-component-table/table.story.js

```diff
@@ -0,0 +1,40 @@
+import { storiesOf } from '@storybook/html';
+import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import withCode from '@ecl-twig/storybook-addon-code';
+
+import table from './table.html.twig';
+import { dataDefault, dataMulti } from './demo/data';
+import docs from './README.md';
```

Comment:
either import the documentation from `./docs/table.md`, or copy the content from `./docs/table.md` to `./README.md` ;)